src/toil/cwl/cwltoil.py

```diff
@@ -1412,6 +1412,13 @@ def remove_unprocessed_secondary_files(unfiltered_secondary_files: dict) -> list
     return final_secondary_files
 
 
+def determine_load_listing(tool):
```

Comment:
There's no docstring here, so how can I tell if this code does what it's supposed to do?

Reply:
Fixed!